site/docs/tutorial/examples.md

```diff
@@ -0,0 +1,12 @@
+# Code samples and examples
+
+It's very useful to have examples of how APIs are used; both to get information on teh best practicses for using APIs and ideas of how they can be used. 
```

Comment:
teh

Reply:
thanks @jacques-n   when the PR in the substrait-java is merged.. I'll update this one